src/client/opamClient.ml

```diff
@@ -1613,8 +1614,13 @@ let check_for_sys_packages config system_packages =
         |> OpamVariable.Map.of_list
       in
       (*Lazy.force header;*)
-      OpamSolution.print_depext_msg set;
-      OpamSolution.install_sys_packages ~confirm:true env config missing ()
+      OpamSolution.print_depext_msg (available, missing);
+      let _ : _ option =
+        OpamSolution.install_sys_packages
```

Comment:
why does a "check" need to install packages?

Reply:
I might have to defer to dra27 on that one :-) It was added in this commit: https://github.com/ocaml/opam/commit/fa4d39f178253db70e1e33baf4ed48357bcc5560#diff-6426799f367266b2a65325c870931b51df35852e12ddbe8580e74f4eeeb58177R1565-R1580